Gemfile

```
@@ -1,12 +1,12 @@
 source 'https://rubygems.org'
 
 # frameworks
-gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
+gem 'rails', '~> 5.2.Z', '>= 5.2.8.1'
```

This is enough:

```suggestion
gem 'rails', '~> 5.2.8'
```

This allows to select any 5.2.x version starting from 5.2.8 and above.